wiki/en/Installation-for-iOS.md

```diff
@@ -38,7 +40,7 @@ If you own a Mac and have Xcode installed, compiling Jamulus from source might b
 
 ## Feedback and development
 
-iOS support isn't mature yet and far from ready to be published on the AppStore. Feel free to help out:
+iOS support isn't mature and not ready to be published on the AppStore. Feel free to help out:
```

Comment:
```suggestion
The view of the Jamulus development teamis that the iOS version is not sufficiently mature to be published on the AppStore. Feel free to help out:
```

Reply:
Why not "our" view?

Reply:
That would be shorter :-)

Reply:
So?
```suggestion
Our view is that the iOS version is not sufficiently mature to be published on the AppStore. Feel free to help out:
```
Is it clear to the reader to whom "our" refers?

Reply:
I think so. I think I've seen other F/LOSS projects using "our" to refer to the people who maintain the project.